wled00/FX_fcn.cpp

```diff
@@ -1376,15 +1389,16 @@ void WS2812FX::service() {
   _triggered = false;
 
   #ifdef WLED_DEBUG
-  if (millis() - nowUp > _frametime) DEBUG_PRINTF_P(PSTR("Slow effects %u/%d.\n"), (unsigned)(millis()-nowUp), (int)_frametime);
+  if (millis() - nowUp > _frametime*2) DEBUG_PRINTF_P(PSTR("Slow effects %u/%d.\n"), (unsigned)(millis()-nowUp), (int)_frametime);
```

Comment:
I do not agree with this.
Reasoning: If effect calculations took more than a single frame, the effects are slow. Same below.

Reply:
The problem I wanted to solve is - when (example) you've updated drawLine and want to test performance, i'd say use "unlimited fps mode". But with debug enabled you'll get spammed on serial with "slow strip/slow effect" messages, so measurement get meaningless.

Another possibility would be to suppress the debug message by `if (getTagetFps() < FPS_UNLIMTED) ...`

Reply:
+1 for the explicit test.

Reply:
explicit test added.